packages/opentelemetry-exporter-stackdriver-trace/src/transform.ts

```diff
@@ -0,0 +1,130 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { hrTimeToTimeStamp, VERSION } from '@opentelemetry/core';
+import { ReadableSpan } from '@opentelemetry/tracing';
+import * as ot from '@opentelemetry/types';
+import {
+  AttributeMap,
+  Attributes,
+  AttributeValue,
+  Link,
+  LinkType,
+  Span,
+  TruncatableString,
+} from './types';
+
+const AGENT_LABEL_KEY = 'g.co/agent';
+const AGENT_LABEL_VALUE = `opentelemetry-version [${VERSION}]`;
+
+export function getReadableSpanTransformer(
+  projectId: string,
+  serviceName: string
+): (span: ReadableSpan) => Span {
+  return span => {
+    const attributes = transformAttributes(span.attributes, {
+      project_id: projectId,
+      service: serviceName,
+      [AGENT_LABEL_KEY]: AGENT_LABEL_VALUE,
+    });
+
+    const out: Span = {
+      attributes,
+      displayName: stringToTruncatableString(span.name),
+      links: {
+        link: span.links.map(transformLink),
+      },
+      endTime: hrTimeToTimeStamp(span.endTime),
+      startTime: hrTimeToTimeStamp(span.startTime),
+      name: `projects/${projectId}/traces/${span.spanContext.traceId}/spans/${span.spanContext.spanId}`,
+      spanId: span.spanContext.spanId,
+      status: span.status,
```

Comment:
make sure failed spans work

Reply:
@mayurkale22 i'm not overly familiar with stackdriver but it doesn't look like failed spans have any visual indication they failed. is this right?

Reply:
Yes, this is right. I think in **Summary** section you will be able to see the status info and add the trace filter on status field.

Reply:
I tried running basic tracer example, with stack driver exporter. This is how status info looks like : 
<img width="315" alt="Screen Shot 2020-01-02 at 10 42 32 AM" src="https://user-images.githubusercontent.com/6525361/71685358-e884c080-2d4c-11ea-9dff-565b20d545c4.png">